### ISSUE TITLE
When doing the training on japanese+english dataset I get the following results

### ISSUE THREAD

**Issue by mariembenslama** [Sierkinhane/crnn_chinese_characters_rec#81]
Start val
U                    => U                   , gt: MNOPQRSTUV

And all test accuracies are 0.

**Comment by Sierkinhane** [comment]
Can you share your train loss.

**Comment by mariembenslama** [comment]
![Sans titre1](https://user-images.githubusercontent.com/49246817/56117969-30667c00-5f61-11e9-9dba-abab4a8579fa.png)
The 1st global lose was 13.214795, then each time it takes only one example like shown in the screenshot, one pred character and tells that the gt is totally different and show a 0.000 accuracy.

The CTC I'm using is from this link: https://github.com/SeanNaren/warp-ctc.git

**Comment by mariembenslama** [comment]
The loss even reachs these values but still prints the wrong prediction.
![Sans titre1](https://user-images.githubusercontent.com/49246817/56119397-f64aa980-5f63-11e9-9753-ab644f3a254e.png)

**Comment by Sierkinhane** [comment]
Networks overfit your training data. You should get more data.

**Comment by mariembenslama** [comment]
I used the data generator and I have 1000 for training and 400 for test, do you think it's not enough?
How much should I add?

**Comment by mariembenslama** [comment]
Alright I added about 10000 samples for training and 5000 for test and here are the results
(same results)
![Sans titre1](https://user-images.githubusercontent.com/49246817/56121350-d87f4380-5f67-11e9-9b45-4b8e6c4af952.png)

I'm just wondering why is it reading only one character in the pred text

**Comment by Sierkinhane** [comment]
I firstly trained on 3.6m synthetic  chinese character data, and then just used synthetic data generated by data_generator for finetune. Therefore, I can't assure that simple data generated by data_generator has robust ability to train a new model. Maybe you should get another data within complexity and diversity.

**Comment by mariembs01** [comment]
Thanks and sorry for the trouble, however, I wrote a small program that read from the lmdb file and I got the following results: 

b\image000001, 0
b\image000002, 0
........

I guess that it reads emtpy values which explains the results I got before 

Start val
U => U , gt: MNOPQRSTUV
0
100
.....

Do you think that would be the problem? I guess that the data_generator you wrote is really good and it's a shame to throw it away.

**Comment by mariembenslama** [comment]
Thanks and sorry for the trouble, however, I wrote a small program that read from the lmdb file and I got the following results:

b\image000001, 0
b\image000002, 0
........

I guess that it reads emtpy values which explains the results I got before

Start val
U => U , gt: MNOPQRSTUV
0
100
.....

Do you think that would be the problem? I guess that the data_generator you wrote is really good and it's a shame to throw it away.

**Comment by mariembenslama** [comment]
When I read and print from the lmdb file (dataset.py) I get the following results:
![Sans titre1](https://user-images.githubusercontent.com/49246817/56198973-daaed400-6033-11e9-9b44-d7646621e729.png)
I guess the whole problem is because of the lmdb file.

**Comment by mariembenslama** [comment]
@CS2JOONGKI about you saying you are training your model for half a day, U can use google collaboratory and copy the project using !git clone link_of_the_project and while training u add the gpu --cuda for fast trainung in less than 20 seconds.

Just for info.

**Comment by jacoc** [comment]
> Start val
> U => U , gt: MNOPQRSTUV
> 
> And all test accuracies are 0.

Hi, I got the same result,  Have you solved it?

**Comment by mariembenslama** [comment]
Hello, yes: Check Holmeyoung Crnn pytorch project, he solved it there :)

**Comment by jacoc** [comment]
> Hello, yes: Check Holmeyoung Crnn pytorch project, he solved it there :)

Thank you for your answer, and it works.
